src/nrepl/repl-start.ts

```diff
@@ -139,16 +139,19 @@ async function extractBundledFiles(
 ) {
     await Promise.all(
         docNames.map(async (docName) => {
-            let docUri: vscode.Uri;
-            docUri = vscode.Uri.file(path.join(storageUri.fsPath, docName));
+            const docUri: vscode.Uri = vscode.Uri.file(
+                path.join(storageUri.fsPath, docName)
+            );
             const templateUri = vscode.Uri.file(
                 path.join(context.extensionPath, TEMPLATES_SUB_DIR, docName)
             );
             try {
                 await vscode.workspace.fs.copy(templateUri, docUri, {
                     overwrite: true,
                 });
-            } catch {}
+            } catch {
+                // continue regardless of error
+            }
```

Comment:
I like these. Will have to revisit and see if are missing some error handling here. The jack-in process is a bit flaky at times..

Reply:
Yeah, catching and not at least reporting it is usually a mistake. I didn't want to introduce new behavior though.